cmd/sidecars/network-passt-binding/server/server.go

```diff
@@ -77,10 +77,11 @@ func (s V1alpha3Server) OnDefineDomain(_ context.Context, params *hooksV1alpha3.
 
 	useVirtioTransitional := vmi.Spec.Domain.Devices.UseVirtioTransitional != nil && *vmi.Spec.Domain.Devices.UseVirtioTransitional
 
-	const istioInjectAnnotation = "sidecar.istio.io/inject"
+	const InjectSidecarAnnotation = "sidecar.istio.io/inject"
+
 	istioProxyInjectionEnabled := false
-	if val, ok := vmi.GetAnnotations()[istioInjectAnnotation]; ok {
-		istioProxyInjectionEnabled = strings.ToLower(val) == "true"
+	if val, ok := vmi.GetAnnotations()[InjectSidecarAnnotation]; ok {
+		istioProxyInjectionEnabled = strings.EqualFold(val, "true")
```

Comment:
Please consider deferring this change to a follow-up, or at least to a different commit.
In case you decide to keep it - please note that there is no need to export `InjectSidecarAnnotation`.